feat: Add @chydeepak7 as a contributor

chydeepak7 opened the pull request:
<!-- Please fill in all areas in this PR form. Incomplete PRs will be marked invalid and may be closed. -->

## Description
This PR add @chydeepak7 as a contributor

<!--
Please do not leave this blank
This PR adds <github-username> as a contributor.
-->

## What type of PR is this? (check all applicable)

- [x] 🤝 Add a contributor
- [x] 📝 Documentation Update

## Related Issues
Fixes #337 

<!-- 
Please use this format link issue numbers: Closes #123
More information: https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword 
-->

## Contributors Checklist

### I've read through the [Getting Started](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=getting-started) section.

- [x] ✅ Yes
- [ ] ❌ Not yet

### Have you run `npm run contributors:generate` to generate your profile and the badge on the README?

- [x] ✅ Yes
- [ ] ❌ No

## Added to documentation?

- [x] 📜 README.md
- [ ] 🙅 no documentation needed

## Screenshot (Required for PR Review)
![image](https://github.com/open-sauced/guestbook/assets/144728684/b818cd6d-fcab-4431-bdf6-84f0925b0671)


<!-- Please provide a screenshot of your profile being generated on the README. This ensures that you ran the `npm run contributors:generate` command, as mentioned in the previous question, which makes it easier for the maintainers to review PRs. All PRs without screenshots will be automatically rejected-->

## [optional] What GIF best describes this PR or how it makes you feel?

<!-- note: PRs with deleted sections will be marked invalid -->

<!--
  For Work In Progress Pull Requests, please use the Draft PR feature,
  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.

  For a timely review/response, please avoid force-pushing additional
  commits if your PR already received reviews or comments.

  Before submitting a Pull Request, please ensure you've done the following:
  - 📖 Read the Open Sauced Contributing Guide: https://github.com/open-sauced/.github/blob/main/CONTRIBUTING.md.
  - 📖 Read the Open Sauced Code of Conduct: https://github.com/open-sauced/.github/blob/main/CODE_OF_CONDUCT.md.
  - 👷‍♀️ Create small PRs. In most cases, this will be possible.
  - 📝 Use descriptive commit messages.
  - 📗 Update any related documentation and include any relevant screenshots.
-->

adiati98 reviewed (changes requested):
Hey @chydeepak7,

Thank you for your contribution! ✨ 

There are a few things here:

- There are failed checks that you can [read here](https://github.com/open-sauced/guestbook/pull/339#pullrequestreview-2081237187). Please read the bot messages and fix the problems until all check pass.
- You opened two PRs. Please close one that you don't use because they are duplication.
- **Very Important:**  You [last commit](https://github.com/open-sauced/guestbook/pull/339/commits/76250a08d648d6c14aa8d5f985cd8bb5cf25b572) deleted the `README.md` and modify the `package-lock.json`. This action will break the production. Please revert the commit. You can read [this blog post](https://dev.to/github/how-to-undo-pushed-commits-with-git-2pe6) to do so.

chydeepak7 commented:
Hello @adiati98 ,
I have deleted the duplicated PR and also reverted the last commit. Can you please check if I have done something wrong.
And as for the bot message I am not able to understand what its saying can you give me a clue of what I should do?

chydeepak7 commented:
package-lock.json how do i revert it I dont know in which commit i changed it

adiati98 commented:
@chydeepak7, look at the last point in my previous comment. I link your commit there.

adiati98 commented:
As for the only bot message left, it's saying that you modified the `package-lock.json`, and a maintainer should carefully review it.

In this case, we (maintainers) don't want to modify this file at all. That's why you need to revert the file. 🙂

chydeepak7 commented:
I think all the checks have passed now.

adiati98 commented on README.md:
```diff
@@ -36,7 +42,11 @@ All contributors are required to abide by our [Code of Conduct](https://github.c
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+<<<<<<< HEAD
 [![All Contributors](https://img.shields.io/badge/all_contributors-141-orange.svg?style=flat-square)](#contributors-)
+=======
+[![All Contributors](https://img.shields.io/badge/all_contributors-142-orange.svg?style=flat-square)](#contributors-)
+>>>>>>> parent of 76250a0 (some minor changes)
```
There's a duplication for the badge here. You need to remove line 47-49.

adiati98 reviewed (changes requested):
Hey @chydeepak7,

When you resolve conflicts, you need to remove the `<<<`, `===`, and `>>>`.

After you address all the reviews, don't forget run the command to generate your profile again in your terminal because now the states are change.

chydeepak7 commented on README.md:
```diff
@@ -1,3 +1,9 @@
+<<<<<<< HEAD
+<<<<<<< HEAD
+ma
+=======
+=======
+>>>>>>> parent of 76250a0 (some minor changes)
```
I checked my README.md but there are no such changes as shown above.

chydeepak7 replied:
I tried in github codespace and those lines are showing. But in my vs code those lines are absent. Why is that??

adiati98 commented:
Hey @chydeepak7 
You haven't run the generate command. 

Please address all the suggestions before requesting for review. 🙂

chydeepak7 commented:
> Hey @chydeepak7 You haven't run the generate command.
> 
> Please address all the suggestions before requesting for review. 🙂

I just did it right now.

adiati98 commented:
@chydeepak7 I see that the badge is not increasing.
Have you run the `npm run contributors:generate` command in your terminal?

chydeepak7 commented:
> @chydeepak7 I see that the badge is missing instead of increasing. Have you run the `npm run contributors:generate` command in your terminal?

yes

adiati98 reviewed (approved):
Well done, @chydeepak7! ✨ 

If you haven't, you can join our community [Discord](https://discord.com/invite/U2peSNf23P).

![git merge GIF](https://media.giphy.com/media/cFkiFMDg3iFoI/giphy.gif)

chydeepak7 commented:
Thank you for your time and guidance
